src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScript.java

```diff
@@ -91,6 +91,16 @@ public CpsScript() throws IOException {
     @Override
     public final Object invokeMethod(String name, Object args) {
         // TODO probably better to call super method and only proceed here incase of MissingMethodException:
```

Comment:
The comment outlines what I suspect would be the cleaner fix.

Reply:
(FWIW. Fine to keep kicking that can down the road, so long as we have test coverage for the stuff we think works.)